plasmapy/formulary/tests/test_relativity.py

```diff
@@ -80,3 +86,221 @@ def test_relativistic_energy():
 
     with pytest.raises(ValueError):
         relativistic_energy(-m, v)
+
+
+proton_at_half_c_inputs = [
+    ("v_over_c", 0.5),
+    ("velocity", 0.5 * c),
+    ("lorentz_factor", 1.1547005383792517),
+    ("total_energy", 1.7358354725115025e-10 * u.J),
+    ("kinetic_energy", 2.3255785652637692e-11 * u.J),
+    ("momentum", 2.8950619440057805e-19 * u.kg * u.m / u.s),
+]
+
+
+@pytest.mark.parametrize("attr, expected", proton_at_half_c_inputs)
+@pytest.mark.parametrize("parameter, argument", proton_at_half_c_inputs)
+def test_relativistic_body(parameter, argument, attr, expected):
```

Comment:
One of the parametrizations is for the velocity-like arguments to `RelativisticBody` used during instantiation, and the other is to make sure the attributes of the resulting `RelativisticBody` instance are correct.  In one run we might create a `RelativisticBody` using `v_over_c` (first parametrization), and then check `RelativisticBody.total_energy` (second parametrization).

Thanks for catching this!  I updated the docstring too since it wasn't clear enough.